src/javascripts/helpers/data/developers.js

```diff
@@ -166,27 +166,27 @@ const developers = [
     github: 'https://github.com/mariabrock',
     website: 'www.mariabrock.dev',
     linkedin: 'https://www.linkedin.com/in/maria-brock/',
-    email: 'mariabrock414@gmail.com',
+    email: 'mailto:mariabrock414@gmail.com',
     isHired: false,
   },
   {
     picture: 'src/images/headshots/MonicaDjunaidi.jpg',
     name: 'Monica Djunaidi',
     resume: 'src/resumes/MonicaDjunaidi_Resume.pdf',
-    github: 'github.com/djunaim',
+    github: 'https://github.com/djunaim',
     website: 'personal-website65796.firebaseapp.com',
-    linkedin: 'linkedin.com/in/monicadjunaidi/',
-    email: 'modjun12@gmail.com',
+    linkedin: 'https://linkedin.com/in/monicadjunaidi/',
+    email: 'mailto:modjun12@gmail.com',
     isHired: false,
   },
   {
     picture: 'src/images/headshots/RandyTate.jpg',
     name: 'Randy Tate',
     resume: 'src/resumes/RandyTate_Resume.pdf',
     github: 'https://github.com/rtate2',
-    website: 'randytatejr.com',
-    linkedin: 'linkedin.com/in/randytatejr',
-    email: 'randytatejr@gmail.com',
+    website: 'https://randytatejr.com',
```

Comment:
https technically works here, but it throws up a warning screen that the site is not private. I'd recommend changing this one to http://randytatejr.com so the user can skip that screen